src/App.jsx

```diff
@@ -14,33 +16,77 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button type="button" className="button is-info is-light">
-        Sort alphabetically
-      </button>
+export const App = () => {
+  const [sortField, setSortField] = useState('');
```

Comment:
Consider initializing `sortField` with a meaningful default value, such as 'alphabet' or 'length', to avoid unexpected behavior when the component first renders.